content/projects/nodejs/sql/_index.md

```diff
@@ -5,6 +5,7 @@ prerequisites:
   hard: ["projects/sql", "workshops/node-and-sql/"]
   soft: []
 tags: ["node", "sql"]
+story_points: 3
```

Comment:
I would give this a story point of 5,because normally recruits have to familiarize themselves and understand with docker and postgresql which takes a bit of effort and time.